README.md

```diff
@@ -1,2 +1,84 @@
 # rokt-demo-ios
-Rokt iOS Demo Application
+Rokt Demo application is a sample app built to showcase Rokt SDK functionality.
+
+## Resident Experts
+- Danial Motahari - danial.motahari@rokt.com
+
+## Requirements
+
+XCode 13 and above. Project is configured to run on iOS 13.0 and above.
+
+## Project architecture
+
+This project is implemented based on MVVM pattern with SwiftUI. 
+MVVM stands for Model-View-ViewModel, an architecture pattern that is structured to separate program logic and user interface controls. It has Model, View and ViewModel separation. 
+
+This project contains UI, Model, Services, Utils and Resources groups.
+
+UI: View and ViewModels are located in the UI Group.
+
+Services & Model: Services represent the network logics of the application and model represents the data.
+
+Utils: are shared between Groups of the application.
+
+Resources: All the other resources of the application is located in Resources. 
+
+![UI diagram](diagram.png)
+
+## Where to start
```

Comment:
start... looking? code? I am not sure what is the purpose of this section

Reply:
I thought it is a good practice to point to a starting point of the application. As we are not using a VIPER design pattern that has a specific layer for routing, `where to start` is a good point specially for grad engineers in NYC. 
However, I would remove it.